contracts/anvil/utils.sh

```diff
@@ -27,13 +27,15 @@ start_anvil_docker() {
     DUMP_STATE_VOLUME_DOCKER_ARG=$([[ -z $DUMP_STATE_FILE ]] && echo "" || echo "-v $DUMP_STATE_FILE:/dump-state.json")
     LOAD_STATE_ANVIL_ARG=$([[ -z $LOAD_STATE_FILE ]] && echo "" || echo "--load-state /load-state.json")
     DUMP_STATE_ANVIL_ARG=$([[ -z $DUMP_STATE_FILE ]] && echo "" || echo "--dump-state /dump-state.json")
+    # temp fix since one of the middleware contract's is greater than default code size limit
+    CODE_SIZE_ARG="--code-size-limit 30000"
```

Comment:
iirc the issue is with the new `DeployMockAvsRegistries.s.sol`, I refactored it [here](https://github.com/Layr-Labs/eigensdk-go/blob/2e5ef1f871760bbe2b8019d39edc75f23d406e53/contracts/script/DeployMockAvsRegistries.s.sol) if you want to bring it in

Reply:
hmm interesting - from what I remember it was one of the middleware contracts but will remove it now and bring in the refactor

Reply:
done! thanks for the refactor!